translations/README.de.md

```diff
@@ -0,0 +1,92 @@
+# GitHub Resume Generator
```

Comment:
Is there any options in German ?

Reply:
> Is there any options in German ?

If you see all the translations didn't change this title

Reply:
I wouldnt translate (Project-)Names